pkg/tsdb/Magefile.go

```diff
@@ -0,0 +1,112 @@
+//go:build mage
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	build "github.com/grafana/grafana-plugin-sdk-go/build"
+)
+
+func find(dir string, name string) ([]string, error) {
+	files := []string{}
+	err := filepath.Walk(dir, func(path string, f os.FileInfo, err error) error {
+		if filepath.Base(path) == name {
```

Comment:
Nit: You have to handle `err` inside the `WalkFunc` here:

```suggestion
		if err != nil {
			return err
		}
		if filepath.Base(path) == name {
```